packages/shared/providers.yaml

```diff
@@ -1124,7 +1126,7 @@ gong-oauth:
     disable_pkce: true
     token_request_auth_method: basic
     proxy:
-        base_url: https://api.gong.io
+        base_url: ${connectionConfig.api_base_url_for_customer} || https://api.gong.io
```

Comment:
@bodinsamuel unless I missed something this won’t work without https://github.com/NangoHQ/nango/pull/1873?

Reply:
ah wasn't aware of this, thanks 👌🏻